frontend/src/main/scala/bloop/engine/tasks/compilation/CompileGraph.scala

```diff
@@ -389,14 +389,15 @@ object CompileGraph {
       PartialFailure(bundle.project, FailedOrCancelledPromise, Task.now(results))
     }
 
-    def loop(dag: Dag[Project], isBestEffortDep: Boolean): CompileTraversal = {
+    def loop(dag: Dag[Project]): CompileTraversal = {
       tasks.get(dag) match {
         case Some(task) => task
         case None =>
           val task: Task[Dag[PartialCompileResult]] = dag match {
             case Leaf(project) =>
               val bundleInputs = BundleInputs(project, dag, Map.empty)
               setupAndDeduplicate(client, bundleInputs, computeBundle) { bundle =>
+                val isBestEffortDep = false
```

Comment:
Why do we need it here? This is the only place we invoke `compile` so isBestEffortDep is always false. Shouldn't we remove it everywhere?

Reply:
It used in line 466/467 (I don't think I can link there as part of the review), where we set it if the dependency was unsuccessful and had betasty artifacts. I left this redundant `val isBestEffortDep = false` as I thought having a false with no context in the lambda would be annoying (but I can change that of course)

Reply:
Ach, alright I haven't noticed that. Thanks for explaning!